kurl-installer.yaml

```diff
@@ -0,0 +1,29 @@
+apiversion: cluster.kurl.sh/v1beta1
+kind: Installer
+metadata:
+  name: ""
+spec:
+  contour:
+    version: latest
+  docker:
+    version: latest
+  kotsadm:
+    version: latest
+  kubernetes:
+    version: latest
+  prometheus:
+    version: latest
+  registry:
+    version: latest
+  velero:
+    version: latest
+  minio:
+    version: latest
+  openebs:
+    version: latest
+    namespace: "openebs"
+    isLocalPVEnabled: true
+    localPVStorageClassName: default
```

Comment:
@aj-jester can we add the flag to disable Cstore as well?

Reply:
@dexhorthy Confirmed with @areed cstor is disabled by default 😄

Reply:
oh. cool. nice! TIL